posthog/warehouse/models/external_data_job.py

```diff
@@ -56,6 +63,19 @@ def get_external_data_job(job_id: UUID) -> ExternalDataJob:
     ).get(pk=job_id)
 
 
+@database_sync_to_async
+def aget_external_data_jobs_by_schema_id(schema_id: UUID) -> list[ExternalDataJob]:
+    from posthog.warehouse.models import ExternalDataSchema
+
+    return list(
+        ExternalDataJob.objects.prefetch_related(
+            "pipeline", Prefetch("schema", queryset=ExternalDataSchema.objects.prefetch_related("source"))
+        )
+        .filter(schema_id=schema_id)
+        .all()
+    )
```

Comment:
We wanna delete data from all jobs except for the latest - someone may have synced their data and then turned off the jobs, and so we don't wanna delete that folder (even though it's using the old folder path)

Reply:
ah yep, good catch